src/Network/FKOAuth1.php

```diff
@@ -0,0 +1,77 @@
+<?php
+/**
+ * @file src/Protocol/OAuth1.php
+ */
+namespace Friendica\Network;
+
+use Friendica\App;
+use Friendica\Core\PConfig;
+use Friendica\Core\System;
+use Friendica\Database\DBM;
+use Friendica\Network\FKOAuthDataStore;
+use dba;
+
+require_once "library/OAuth1.php";
+require_once "include/plugin.php";
+
+/**
+ * @brief OAuth protocol
+ */
+class FKOAuth1 extends OAuthServer
+{
+	/**
+	 * @brief Constructor
+	 */
+	public function __construct()
+	{
+		parent::__construct(new FKOAuthDataStore());
+		$this->add_signature_method(new OAuthSignatureMethod_PLAINTEXT());
+		$this->add_signature_method(new OAuthSignatureMethod_HMAC_SHA1());
+	}
+
+	/**
+	 * @param string $uid user id
+	 * @return void
+	 */
+	public static function loginUser($uid)
+	{
+		logger("FKOAuth1::loginUser $uid");
+		$a = get_app();
+		$r = dba::select('user', array(), array('uid' => $uid, 'blocked' => 0, 'account_expired' => 0, 'account_removed' => 0, 'verified' => 1), array('limit' => 1));
```

Comment:
You could assign ```$record``` directly, then you wouldn't need to assign this variable some lines later.